frontend/src/components/debug/UserDebug.vue

```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { Get, Post, Delete, Patch } from '@/services/requests';
+
+import { storeToRefs } from 'pinia';
+import { useUserStore } from '@/stores/user';
+import type { User } from '@/models/user.model';
+import type { Input, InputStore } from '@/stores/input';
+
+// Props
+const props = defineProps<{
+  title: string;
+  inputStore: InputStore;
+  input: Input;
+}>();
+
+// Request url to API
+const baseUrl = 'http://localhost:3000/users';
```

Comment:
It didn't work: submitting a new user from frontend did nothing. If I understand correctly, and `baseUrl` is the address of the API, then the port should `4000`.